### PR TITLE
Id 33 user location on the map

### PR DESCRIPTION
**Pull request by wojtek717**

### CODE REVIEW

**Comment by wojtek717** [comment]
[User location on Map](https://trello.com/c/mvHNflqx/33-user-location-on-map)

File: kpz-mpk/kpz-mpk/Map/LocationService.swift [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    private let locationManager = CLLocationManager()
+    private(set) var userLocation: CLLocationCoordinate2D?
+    private var didUpdateInitialLocation = false
+
+    // MARK: - Initializers
+    public override init() {
+        super.init()
+
+        locationManager.delegate = self
+      locationManager.startUpdatingLocation()
```
Comment (leszko11) [inline kpz-mpk/kpz-mpk/Map/LocationService.swift]: missing intendation

File: kpz-mpk/kpz-mpk/Map/ViewModels/MapViewModel.swift [hunk 2]
```diff
@@ -11,20 +11,35 @@ import MapKit
 
 final class MapViewModel {
   private weak var presenter: MapViewControllerPresenter?
-  //hardcoded for now
-  private var userLocation: CLLocationCoordinate2D = CLLocationCoordinate2D(latitude: 51.111510, longitude: 17.059737)
+  var locationService = LocationService()
```
Comment (leszko11) [inline kpz-mpk/kpz-mpk/Map/ViewModels/MapViewModel.swift]: plz make it `let`

File: kpz-mpk/kpz-mpk/Map/LocationService.swift (hunk 1 above)
Comment (wojtek717) [reply kpz-mpk/kpz-mpk/Map/LocationService.swift]: oh, i have missed that

**Pull request merged**